src/dotnet/APIView/APIView/CodeLine.cs

```diff
@@ -45,7 +48,7 @@ public override string ToString()
 
         public bool Equals(CodeLine other)
         {
-            return DisplayString == other.DisplayString && ElementId == other.ElementId;
+            return DisplayString == other.DisplayString && ElementId == other.ElementId && other.IsHiddenApi == IsHiddenApi;
```

Comment:
My assumption is that this is safe because all out-dated codefiles will be updated in the background to the latest version. If we think this is too dangerous I can make the new property nullable and add a safeguard such that if either is null, we don't consider it a difference. This way old files containing hidden APIs will not trigger a diff against new files.